Products/ZenUI3/browser/eventconsole/export.py

```diff
@@ -75,11 +77,18 @@ def csv(self, response, archive, **params):
         response.setHeader('Content-Disposition', 'attachment; filename=events.csv')
         from csv import writer
         writer = writer(response)
-
+        events = [dict((k, v) for k, v in evt.iteritems() if v) for _, evt in
+                  self._query(archive, **params)]
```

Comment:
An alternate way to create the dictionary is to use a dictionary comprehension, like so:
```
events = [
    {k: v for k, v in evt.iteritems() if v}
    for _, evt in self._query(archive, **params)
]
```